lib/contestant.rb

```diff
@@ -0,0 +1,28 @@
+class Contestant
+  attr_reader :age, :state_of_residence, :spending_money, :game_interests
+
+  def initialize(contestants)
+    @contestant = contestants
+    @age = contestants[:age]
+    @state_of_residence = contestants[:state_of_residence]
+    @spending_money = contestants[:spending_money]
+    @game_interests = []
+  end
+
+  def full_name
+    @contestant[:first_name] + " " +  @contestant[:last_name]
+  end
+
+  def out_of_state?
+    if @contestant[:state_of_residence] == "CO"
```

Comment:
Once again, no need for control flow, and no need to store all of the contestant info as `@contestant`:
`@state_of_residence != "CO"`